content/05.treat.md

```diff
@@ -180,28 +180,24 @@ However, in the long term, atomic convolutions may ultimately overtake grid-base
 
 Another approach to *de novo* design is to train character-based RNNs on large collections of molecules, for example, ChEMBL [@doi:10.1093/nar/gkr777], to first obtain a generic generative model for drug-like compounds [@tag:Segler2017_drug_design].
-These generative models successfully learn the grammar of compound representations, with 94% [@tag:Olivecrona2017_drug_design] or nearly 98% [@tag:Segler2017_drug_design] of generated SMILES corresponding to valid molecular structures.
-The initial RNN is then fine-tuned to generate molecules that are likely to be active against a specific target by either continuing training on a small set of positive examples [@tag:Segler2017_drug_design] or adopting reinforcement learning strategies [@tag:Olivecrona2017_drug_design; @arxiv:1611.02796].
-Both the fine-tuning and reinforcement learning approaches can rediscover known, held-out active molecules.
-The great flexibility of neural networks, and progress in generative models offers many opportunities for deep architectures in *de novo* design (e.g. the adaptation of GANs for molecules).
+These generative models successfully learn the grammar of compound representations, with 94% [@tag:Olivecrona2017_drug_design] or nearly 98% [@tag:Segler2017_drug_design] of generated SMILES corresponding to valid molecular structures. The initial RNN is then fine-tuned to generate molecules that are likely to be active against a specific target by either continuing training on a small set of positive examples [@tag:Segler2017_drug_design] or adopting reinforcement learning strategies [@tag:Olivecrona2017_drug_design; @arxiv:1611.02796]. Both the fine-tuning and reinforcement learning approaches can rediscover known, held-out active molecules.
+
+Reinforcement learning approaches where operations are performed directly on the molecular graph bypass the need to learn the details of SMILES syntax, allowing the model to focus purely on chemistry. Additionally, they seem to require less training data and generate more valid molecules since they are constrained by design only to graph operations which satisfy chemical valiance rules.[@tag:Elton_molecular_design_review] A reinforcement learning agent developed by Zhou et al. demonstrated superior molecular optimization performance on certain easy to compute metrics when compared with other deep learning based approaches such as the Junction Tree VAE, Objective Reinforced Generative Adversarial Network, and Graph Convolutional Policy Network.[@doi:10.1038/s41598-019-47148-x] As another example, Zhavoronkov et al. used generative tensorial reinforcement learning to discover potent inhibitors of discoidin domain receptor 1 (DDR1).[@tag:Zhavoronkov2019_drugs] Their work is unique in that six lead candidates discovered using their approach were synthesized and tested in the lab, with 4/6 achieving some degree of binding to DDR1.[@tag:Zhavoronkov2019_drugs]  
```

Comment:
I will have time to respond to your points and make a number of improvements starting on Friday afternoon. 

Incidently, there has also back and forth recently about the GENTRL reinforcement learning method, with some skepticism from the phareceutical world. ([see the response here and references](https://www.nature.com/articles/s41587-020-0417-3))

Reply:
I read the paper Walters and Murcko and the response paper I linked to, but I'm not sure how much of that needs to go in the review as it gets a bit "in the weeds". The basic critique is that a lot of the molecules being generated are variations on stuff in the training data. They argue for standards such as including the training data used with the paper and a ranking of the closest molecules in the training data, so that novelty can be discerned. 

It may be worth discussing the need for metrics / criteria for scoring generative models or deep learning based systems for optimization more generally..  I discussed it in my review article on deep learning for molecular design (section 3.1). If you can't access the Royal Society of Chemistry version, [its on the arXiv here](https://arxiv.org/pdf/1903.04388.pdf). Note the criteria discussed are more for generation (ie ensuring a large, diverse set), rather than optimization for a specific target.